src/apiCalls.js

```diff
@@ -0,0 +1,32 @@
+const apiKey = process.env.REACT_APP_API_KEY;
```

Comment:
Really nice job hiding your API key here.

Reply:
Yes, this was definitely a challenge trying to figure out at first.